bindings/protocols/modbus/index.html

```diff
@@ -75,23 +75,33 @@
+            For this reason, it has a low level of abstraction and it has built in bit handling capabilities oriented to the direct
             control of the relays and generic contact statuses.
-            The physical layer it is mainly an RS485 differential bus which has less susceptibility to the EMC interference. This Limit
-            the usability of the protocol in short distance networks, typically within a kilometer. 
-            Due to its age the protocol does not implement any safety system, so usually when wide internet access is needed it is
-            encapsulated in a TCP/IP protocol and with Ethernet as a physical layer. Thanks to this encapsulation strategy, the Modbus protocol
-            can reach remote nodes deployed in distant facilities over the internet. Moreover, in the years due to its simplicity and cost-effectiveness 
-            the Modbus protocol becomes a standard all over the world. This fame together with the advancement of the microcontroller/microprocessor led to 
-            a shift on how applications pack the information. Today it is usual to store and read any type of data in the Holdings Registers like bit, bytes, words, float etc.
         </p>
         <p>
-            This document describes how the Web of Things specification can be use to present devices that use the Modbus protocol in a <a href="https://w3c.github.io/wot-architecture/#sec-thing-description">Thing Description</a>. 
-            In particular, the document explain how to create valid URLs and Forms for the different operations that the Modbus protocol can perform. 
+            All Modbus devices operate by exposing a certain set of their coils (bits) and registers (group of 16 bits) over the network.
+            These coils or registers can be read or written to with a simple request-response mechanism.
+            Very typically, registers are also combined to represent data that is larger than 16 bits, i.e. two registers can be used
+            together to represent a 32 bit integer.
+            Thus, a Modbus client can read a set of registers at once and obtain a large value.
+        </p>
+        <p>
+            The physical layer can be RS485 differential bus which has less susceptibility to the EMC interference.
+            This limits the usability of the protocol in short distance networks, typically within a kilometer.
+            However, when Internet access is needed, it is encapsulated in TCP (called Modbus TCP) with Ethernet as the physical layer. 
+            Thanks to this encapsulation strategy, the Modbus protocol can reach remote nodes deployed in distant facilities over the Internet. 
+            Moreover, over the years, due to its simplicity and cost-effectiveness the Modbus protocol became a standard all over the world. 
+            This popularity, together with the advancement of the microcontroller/microprocessor, has led to a shift on how applications pack the information.
```

Comment:
This sentence feels a little bit out of the blue, I think removing 
> Today it is usual to store and read any type of data in the Holdings Registers like bit, bytes, words, float etc.

Decrease clarity.

Reply:
This was removed actually, I don't see it?

Reply:
This was about reintroducing that sentence. I have done that in the latest commit. Resolving.